### DISCUSSION TITLE
What the summary of the Total Net Worth on the Dashboard means

### DISCUSSION THREAD

**Discussion by Dorusto** [moneymanagerex/moneymanagerex#6253] [General]
Hello everyone. I'm trying to understand how is getting the amount in the Reconcilied and the Ballance from the Dashboard. I dont see any connection with my total amount. I did not find anything explained in the help also. Please see the attachement. Some of you can make some light in my dark mind? :) Thank you! 
![Screenshot from 2023-10-20 22-27-14](https://github.com/moneymanagerex/moneymanagerex/assets/19708912/2326e3c3-3280-472a-98be-fe1d4a97ba31)

**Comment by whalley** [comment]
Do you have any closed accounts with non-zero balances?

**Reply by Dorusto** [reply]
I don't think so. I'm still trying to understand how Stock Portofolios works so maybe I did something wrong there? I've put the initial balance in the main account and then I create for each stock a transaction to be able to mach the account balance with the real one. This is the way to do it? I will check all the accounts created, maybe I did some some mistakes in the setup process.
![image](https://github.com/moneymanagerex/moneymanagerex/assets/19708912/ded5d1ec-2f48-4e35-93b4-18b428fe870b)

**Comment by nickallington** [comment]
I have to say I have this issue too. I cannot reconcile the Reconciled total at the top with the cash, credit and bank accounts that are listed on the front page. Also, the Total Net Worth is not equal to the sum of Reconciled, Assets and Stock. Even if I try including unreconciled (pending) transactions, it is out of balance. I am not sure why that would be. I do have multiple currencies, but the exchange rate tables are up to date.

**Comment by georgeef** [comment]
The Reconciled total in Dashboard is the sum of reconciled transactions with:
- several account types which are shown in widgets (left)
- accounts of type Asset or Shares, which are not shown in widgets

Assets and Stocks in widgets are not the same as accounts of type Asset or Shares, resp. So if you have accounts of these two types, and if you have reconciled transactions with these accounts (to check you can open All Transactions and search for each account), then they are also added in the Reconciled total.

**Reply by nickallington** [reply]
OK, that worked. I went through everything and identified a series of rogue transactions, and once they were deleted, the share account transactions marked as reconciled, and the unreconciled (pending) bank transactions subtracted, it added up properly, which is good!

But that does add to my concerns (recorded elsewhere) that work need to be done on the investment portfolio functionality.
Thanks

**Reply by georgeef** [reply]
Could you add a link to the issue/discussion with your concerns?

**Reply by nickallington** [reply]
I can't really, as comments are dotted around the MMEX Discussion forum as well as on here, but after a few bilateral emails I am starting to draft a user requirements document that can be reviewed and enhanced by those that are interested. Once (if...)that gets off the ground there will be a discussion group for it.
